examples/storage_pool.rb

```diff
@@ -31,7 +31,8 @@
   storage_system @storage_system_ip
   data(
     description: "SSD Storage pool - CHEF",
-    isManaged: false
+    isManaged: false,
```

Comment:
Update the license to 2020 at the top of this file.

Reply:
corrected